packages/node_modules/node-red/red.js

```diff
@@ -14,6 +14,16 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  **/
+
+if (process.argv[2] === 'admin') {
+    try {
+        require("node-red-admin")(process.argv.slice(3))
```

Comment:
The only concern, I'd have with this line: If anyone wants to build a single-executable Node-RED binary using https://github.com/vercel/pkg it might not work due to the "dynamic require".

Reply:
That shouldn't be a problem. pkg only fails with package names built at runtime.

However, I wasn't able to get pkg working at all, yet. Probably due to Node-RED's dynamic module and node loading at runtime.

Reply:
What dynamic require are you referring to? `node-red-admin` is declared as a dependency in `package.json` and is required here just like any other module. There is nothing dynamic about it.

The dynamic loading of nodes for the palette is far more likely to be an issue, but that's how NR has worked for 7 years, so nothing new there.

Reply:
Oh yeah, you are right @kuema. In that case it shouldn't have any impact :+1: